OpenRA.Game/Graphics/SpriteRenderer.cs

```diff
@@ -124,7 +124,27 @@ public void DrawSprite(Sprite s, float3 location, PaletteReference pal, float3 s
 		public void DrawSprite(Sprite s, float3 a, float3 b, float3 c, float3 d)
 		{
 			var samplers = SetRenderStateForSprite(s);
-			Util.FastCreateQuad(vertices, a, b, c, d, s, samplers, 0, nv);
+			var tint = new float3(1, 1, 1);
```

Comment:
Would it make sense to add some sort of `const` (private) field here, or is the compiler smart enough to do this already and create a new struct every draw?